app/reduxContent/createDelegate/thunks.js

```diff
@@ -83,45 +87,35 @@ export function createNewAccount(
       publicKeyHash
     );
     const { url } = getSelectedNode(settings, TEZOS);
+    const realKeyStore = keyStore;
+    let realDerivation = '';
 
-    let newAccount;
     if (isLedger) {
-      const newKeyStore = keyStore;
       const { derivation } = getCurrentPath(settings);
-
-      newKeyStore.storeType = 2;
-      newAccount = await sendAccountOriginationOperation(
-        url,
-        newKeyStore,
-        amountInUtez,
-        delegate,
-        true,
-        true,
-        fee,
-        derivation
-      ).catch(err => {
-        const errorObj = { name: err.message, ...err };
-        console.error(errorObj);
-        dispatch(addMessage(errorObj.name, true));
-        return false;
-      });
-    } else {
-      newAccount = await sendAccountOriginationOperation(
-        url,
-        keyStore,
-        amountInUtez,
-        delegate,
-        true,
-        true,
-        fee
-      ).catch(err => {
-        const errorObj = { name: err.message, ...err };
-        console.error(errorObj);
-        dispatch(addMessage(errorObj.name, true));
-        return false;
-      });
+      realDerivation = derivation;
+      realKeyStore.storeType = 2;
     }
 
+    const newAccount = await sendAccountOriginationOperation(
```

Comment:
`sendAccountOriginationOperation` does not take code or storage as a parameter in ConseilJS. We should call `sendContractOriginationOperation` to originate a smart contract